planning/motion_velocity_planner/autoware_motion_velocity_planner/src/node.hpp

```diff
@@ -67,6 +68,8 @@ class MotionVelocityPlannerNode : public rclcpp::Node
   tf2_ros::TransformListener tf_listener_;
 
   // subscriber
+  autoware_utils_rclcpp::InterProcessPollingSubscriber<autoware_planning_msgs::msg::LaneletRoute>
+    route_subscriber_{this, "~/input/route", rclcpp::QoS{1}.transient_local()};
```

Comment:
[nits] align name convention
```suggestion
    sub_route_{this, "~/input/route", rclcpp::QoS{1}.transient_local()};
```

Reply:
resolved https://github.com/autowarefoundation/autoware_core/pull/584/commits/b40fa24cef1d645ba4afbffe5f614da00e1fd6c5